google-cloud-storage/src/main/java/com/google/cloud/storage/BlobInfo.java

```diff
@@ -260,6 +261,22 @@ static CustomerEncryption fromPb(StorageObject.CustomerEncryption customerEncryp
      */
     public abstract Builder setCrc32c(String crc32c);
 
+    /**
+     * Sets the custom time for an object. Once set it can't be unset and only changed to a custom
+     * datetime in the future. If the custom_time must be unset, you must either perform a rewrite
```

Comment:
Don't use "custom_time" in this context, say "custom time" instead. Replace "If the custom_time must be unset," with "To unset the custom time,"

Reply:
done